apps/emqx_auth/src/emqx_authz/emqx_authz_rule_raw.erl

```diff
@@ -98,7 +99,7 @@ validate_rule_topics(RuleRaw) ->
     throw({missing_topic_or_topics, RuleRaw}).
 
 validate_rule_topic(<<"eq ", TopicRaw/binary>>) ->
-    {eq, validate_rule_topic(TopicRaw)};
```

Comment:
meaning?
`<<"eq eq a/#">>` is parsed as `{eq, <<"eq a/#">>}`?

Reply:
Yes, and "eq a/#" is a valid topic